### PR TITLE
Get VRC and verify driver

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

**Comment by iamniting** [comment]
duplicate of https://github.com/kube-storage/volume-replication-operator/pull/23

File: controllers/volumereplication_controller.go
Comment (Madhu-1) [inline controllers/volumereplication_controller.go:83]: question: will this work for clusterScoped resources?

File: controllers/volumereplication_controller.go
Comment (Madhu-1) [inline controllers/volumereplication_controller.go:95]: As the driver name is not matching we should not log or return an error we just need to return success here

**Comment by Madhu-1** [comment]
Merge this after merging #23

**Review by sp98 — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Pull request merged**
